### ISSUE TITLE
[UAC pages] accurate and informative page titles

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#5882]
## Area or Component

`/uac/set-password`
`/uac/set-recovery-question`
`/uac/mfa-select`
`/uac/mfa-sms`
`/uac/mfa-okta`
`/mfa-okta/verify`
`uac/mfa-google-auth`
`/uac/mfa-google-auth/verify`
`/uac/mfa-security-key`
`/uac/mfa-phone`
`/uac/mfa-phone/verify`
`/uac/mfa-email/verify`
`/uac/success`

## Quick Summary

The `<title>`s for these pages are just `SimpleReport' when they should be accurate and informative

### How to Replicate
This didn't get caught in the axe chrome dev tools.

## Accessibility Violation

[Page title best practice from Deque
](https://dequeuniversity.com/checklists/web/page-title)

## Action Requested

Add unique and accurate page titles for each of these flows

## Source Code (optional)

[which file or line is the issue attached to?]: #

**Comment by johanna-skylight** [comment]
@lauraykerr I have used the same text in the headings of each page as the page titles. Below is a summary of how they will look like. When you have a chance can you check if they look correct and let me know if you need me to set something different. Thanks!

<img width="664" alt="Screenshot 2023-06-07 at 10 29 58 AM" src="https://github.com/CDCgov/prime-simplereport/assets/103958711/455b9ce3-a617-454c-a6b4-57c4d708e8c2">

**Comment by johanna-skylight** [comment]
> @johanna-skylight Is it possible to add " | SimpleReport" to the end of all titles? For example, "Verify your security code | SimpleReport". That's our typical convention across the app and site.
> 
> It also looks like there are a few duplicated titles above ("Verify your security code"). I'm not as familiar with these pages, but if you think these accurately reflect what's on there, can we change those to the following:
> 
> * `/mfa-sms/verify`: Verify your security code from text message (SMS) | SimpleReport
> * `/mfa-okta/verify`: Verify your security code from Okta Verify | SimpleReport
> * `/mfa-google-auth/verify`: Verify your security code from Google Authenticator | SimpleReport
> * `/mfa-phone/verify`: Verify your security code from a phone call | SimpleReport
> * `/mfa-email/verify`: Verify your security code from email | SimpleReport

Oh sorry for forgetting to add the small detail but yeah! that '| SimpleReport' will totally be part of all the titles.

As for the verify pages you can check them in [this link](https://60a556a7c807cc0039ec6786-tlaclcphpo.chromatic.com/?path=/story/example-introduction--page). Under the `Account setup` section, all the ones marked with `step 3b`.
<img width="365" alt="Screenshot 2023-06-07 at 12 09 11 PM" src="https://github.com/CDCgov/prime-simplereport/assets/103958711/7f4afb48-5f1b-4d27-b03c-678cf75ea10d">

**Comment by lauraykerr** [comment]
Checked out the "Verify your security code" pages and the suggested titles above seem to match, so should be good to roll with those plus the ones from the page headings (+ " | SimpleReport"). Thanks, @johanna-skylight!
